frontend/src/lib/components/ActivityLog/activityLogLogic.tsx

```diff
@@ -109,11 +109,13 @@ export const activityLogLogic = kea<activityLogLogicType>({
         const onPageChange = (
             searchParams: Record<string, any>,
             hashParams: Record<string, any>,
-            pageScope: ActivityScope
+            pageScope: ActivityScope,
+            forceUsePageParam?: boolean
```

Comment:
Ah, yeah, I'd forgotten about fighting the multiple ways that tabs do or don't update URL and expect paging.

No need to fix that confusion here though :)